SpatialGDK/Source/SpatialGDK/Private/Schema/StaticSymbols.cpp

```diff
@@ -0,0 +1,47 @@
+#include "Schema/ActorGroupMember.h"
+#include "Schema/ActorOwnership.h"
+#include "Schema/ActorSetMember.h"
+#include "Schema/AuthorityIntent.h"
+#include "Schema/DebugComponent.h"
+#include "Schema/GameplayDebuggerComponent.h"
+#include "Schema/Interest.h"
+#include "Schema/MigrationDiagnostic.h"
+#include "Schema/NetOwningClientWorker.h"
+#include "Schema/PlayerSpawner.h"
+#include "Schema/Restricted.h"
+#include "Schema/ServerWorker.h"
+#include "Schema/SnapshotVersionComponent.h"
+#include "Schema/SpatialDebugging.h"
+#include "Schema/SpawnData.h"
+#include "Schema/StandardLibrary.h"
+#include "Schema/Tombstone.h"
+#include "Schema/UnrealMetadata.h"
+
+#if __cplusplus <= 201402L
```

Comment:
Is this a definition of constexpr values in a cpp? I never saw this, neither have I had problems with just declaring+defining a static constexpr in a header file. Is this required?

Reply:
Ah OK I see you've elaborated on this:
> Had to fix an obscure linking error for Clang in C++14 (UE 4.25) : stackoverflow.com/questions/8016780/undefined-reference-to-static-constexpr-char

Reply:
I think this should be commented in code though - I think the next person is going to be just as confused until they decide to go through old PRs.

Reply:
UE 4.25 Linux build fails otherwise. Look at the linked stackoverflow thread if you want more details.
It started breaking once I used them in templates.

Reply:
I'll add a comment to remove it once UE 4.25 is no longer supported.

Reply:
Is this a UE 4.25 problem specifically? I thought this is just clang adhering to the standard for c++14 but nitpicking different rules from msvc.

Does this go away if it's static const instead of constexpr?

Reply:
UE 4.26 uses c++17.
It would go away as well, but then it would not allow us to drop the StaticConstants file when we do not have to support UE 4.25 (plus all the weirdness it might cause because we would need to dllexport the constants)